pym/bob/input.py

```diff
@@ -2504,7 +2533,7 @@ def __parseUserConfig(self, fileName):
             self.__archive = cfg["archive"]
         self.__scmOverrides.extend([ ScmOverride(o) for o in cfg.get("scmOverrides", []) ])
         self.__aliases.update(cfg.get("alias", {}))
-
+        self.__userConfig.update(cfg.get("config", {}))
```

Comment:
I think this is a bit too coarse. This way a config file can override only all switches of `dev` or `build` but not individual ones.